README.md

```diff
@@ -1 +1,89 @@
-# Capstone-Project
\ No newline at end of file
+# Capstone-Project
+
+> This project, the capstone, is made at the end of each module in order to recap all skills we gain during the apprendship period. It helps us to master some skills and made researches about what seems not understanding during the program.
+
+
+## Built With
+
+- HTML, CSS & JAVASCRIPT
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Git/Github
+- VSCode or another equivalent tool
+- Html, Css and Javascript
+
+### Install
+
+- npm install -g @lhci/cli@0.7.x
+- npm install --save-dev hint@6.x
+-  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
```

Comment:
- [ ] Kindly add a step on how the user can run your project locally so your project is documented professionally.